mitodl/micromasters
# asadiqbal08/Migrating from ElasticSearch to OpenSearch

#### What are the relevant tickets?
fixes: https://github.com/mitodl/micromasters/issues/5155

#### What's this PR do?
We migrated from Elasticsearch to OpenSearch

#### How to test ?
- Make sure the env for `OPENSEARCH_INDEX` is set e.g. `micromaster`, also the `OPENSEARCH_DEFAULT_PAGE_SIZE=50`
- Build the container with OpenSearch image
- Execute the `recreate_index` command to check the indexing pattern.
- Other, test the searching via providing some queryparams to OpenSearch searching endpoint `/api/v0/search/`
